doc/source/cluster/running-applications/job-submission/openapi.yml

```diff
@@ -0,0 +1,471 @@
+openapi: 3.1.0
+info:
+  title: Ray Jobs API
+  version: 4.0.0
```

Comment:
The `/api/version` returns `"4"`, not `"4.0.0"`.  Do these need to be kept in sync?